nike_sneaker_store/lib/themes/ns_theme.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:nike_sneaker_store/resources/ns_color.dart';
+
+class NSTheme {
+  NSTheme._();
+
+  static ThemeData lightTheme = ThemeData(
+    brightness: Brightness.light,
```

Comment:
missing config for text theme, text button theme, app bar.
You should read about how to use theme